library/core/src/slice/iter.rs

```diff
@@ -355,6 +355,11 @@ impl<'a, T: 'a, P: FnMut(&T) -> bool> Split<'a, T, P> {
     pub(super) fn new(slice: &'a [T], pred: P) -> Self {
         Self { v: slice, pred, finished: false }
     }
+    /// Returns items which were not yet returned by split.
+    #[unstable(feature = "slice_remainder", issue = "none")]
+    pub fn remainder(&self) -> &'a [T] {
```

Comment:
I think this should be called `as_slice()` instead, since that's the way it's done for `vec::IntoIter` or `slice::Iter`. And then we could also implement `AsRef<[T]>` like they do.

Reply:
I think this would have to be insta-stable, but if that's fine

Reply:
It also matches `str::Split`'s version of remainder. (`as_str()`)

Reply:
> I think this would have to be insta-stable, but if that's fine

New features require a tracking issue. Stabilizing something requires an FCP on a tracking issue so it'll be run by the libs team anyway. To me personally it looks fine and consistent with other Iterators.

Reply:
I agree, this needs to be run past the libs team, especially since there is maybe an opportunity to unify with #77998.

Reply:
Well the issue is that trait implementations are insta-stable, or at least I can't add an unstable implementation. I can make a tracking issue if this would be agreeable to the libs team but I'm not gonna get ahead of myself and do it before approval

Reply:
I don't see an urgent need to add an `AsRef` impl so I'd prefer to avoid doing that without strong motivation, but renaming the method from `remainder` to `as_slice` for consistency with `as_str()` on `str::Split` sounds like a good idea to me.